WhyNot/WhyNot/UNO-102/PeopleViewModel.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  PeopleImagesData.swift
+//  WhyNot
+//
+//  Created by Fernando Santos on 03/11/22.
+//
+
+import Foundation
+import SwiftUI
+
+class PeopleVM: ObservableObject {
+    @Published var maxImagesNumberPossible = 4
+    @Published var images: [UIImage]
```

Comment:
essa galerinha num precisa ser published, já que essas variáveis não são acessadas pela view
```suggestion
    let maxImagesNumberPossible = 4
    let images: [UIImage]
```